server/routes/charge-card.js

```diff
@@ -0,0 +1,29 @@
+var express = require('express');
+var Stripe = require('stripe')
+var router = express.Router();
+
+const stripe = new Stripe('sk_test_51IweHkKvAxvZ5kVetx3UiymBoyjtPixhy4q72vX0lh5MaPQfI9r9zilffbnSnOXRsfbJKbYSNQAkTCB9vPWyTBOO00WrHtvZsQ');
+
+/* Create charges on Stripe. */
+router.post('/', async function (req, res, next) {
+    const paymentDetails = JSON.parse(Object.keys(JSON.parse(JSON.stringify(req.body)))[0])
```

Comment:
I don't really get what's going on here, is there a cleaner way to refactor this?

Reply:
Why do you have to stringify then parse again rather than just just using req.body object? Perhaps you can split this long line into different named variables.